bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -359,7 +359,7 @@ impl Script {
     /// removed. Use `is_op_return` if you want `OP_RETURN` semantics.
     #[deprecated(
         since = "0.32.0",
-        note = "The method is not very useful, you might want `is_op_return`"
+        note = "The method has potentially confusing semantics and is going to be removed, you might want `is_op_return`"
```

Comment:
```suggestion
        note = "The method has potentially confusing semantics and is scheduled for removal, please use `is_op_return`."
```

A suggestion for a more polished looking message.
I don't want to bike shed it. Feel free to ignore it.
Any informative deprecation notice is a good thing.

I think also that this needs to be softwrapped, no?
Anyhow this is handled by the weekly nightly `rustfmt`...

Reply:
Thanks for the review, we only keep deprecated things around for one release so this will be removed (likely as soon as the release comes out). I'm wasn't too fussy with it, also in this case, I believe `is_op_return` might not be what folk always want, that's why the wording is like it is. "scheduled for" is definitely better but since this has two acks I'll just leave it.